src/main/java/com/toamistaketracker/ToaMistake.java

```diff
@@ -84,10 +100,11 @@ public enum ToaMistake {
     private static final String FALLBACK_IMAGE_PATH = "death.png";
 
     private static final int MAX_STACKING_CHAT_MESSAGE_LENGTH = 10;
+    public static final int MAX_MESSAGE_LENGTH = 40;
 
     @Getter
     @NonNull
-    private final String mistakeName;
+    private final String mistakeName, defaultMessage;
```

Comment:
Not a fan of multi-declaration on a single line. Please put each field on its own.